### DISCUSSION TITLE
tag colors & relations

### DISCUSSION THREAD

**Discussion by jdion84** [filamentphp/filament#8997] [Help]
wondering if its just not documented but possible

im interested in having the tags field use a belongstomany relationship, along with a color for each tag

i would have a tag model, and the tag model would have a color field

is this possible with filament natively?

**Comment by leandrocfe** [comment]
You mean, in the Form builder or [Table Builder](https://filamentphp.com/docs/3.x/tables/columns/text#displaying-as-a-badge)?

**Reply by jdion84** [reply]
form builder, i can map the tags in the table builder easily

**Reply by leandrocfe** [reply]
isn't possible I think
